src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -264,14 +265,24 @@ public static void UpdateBackgroundLayerFrame(this UIView view)
 
 			var layer = view.Layer;
 
+			UpdateBackgroundLayerFrame(layer, view.Bounds);
+		}
+
+		static void UpdateBackgroundLayerFrame(CALayer layer, CGRect bounds)
+		{
 			if (layer == null || layer.Sublayers == null || layer.Sublayers.Length == 0)
 				return;
 
 			foreach (var sublayer in layer.Sublayers)
 			{
-				if (sublayer.Name == BackgroundLayerName && sublayer.Frame != view.Bounds)
+				UpdateBackgroundLayerFrame(sublayer, bounds);
```

Comment:
Does this change make the change on this PR no longer necessary?

https://github.com/dotnet/maui/pull/7925/files#diff-cf56310b45ff03f77e3aa0ee6c5d9b717ea28d750ce396f1b50148ccbcbed4f8R138

Reply:
Still is necessary. We are changing the method to search recursively but we still need to invoke it with Frame changes (resize, change orientation etc).